pages/docs/coding-conventions/code-organization.en-US.mdx

```diff
@@ -0,0 +1,140 @@
+# Code organization
+
+- Colocate related files in the same directory. Each component should have its own directory containing all its associated files. This includes:
+  - Component file (e.g., `patient-banner.component.tsx`) - This is a React component.
+  - Test file (e.g., `patient-banner.test.tsx`) - This is a Jest test file.
+  - Stylesheet (e.g., `patient-banner.scss`) - This is a CSS stylesheet.
+  - Resource file (e.g., `patient-banner.resource.ts`) - This is a file that contains data fetching logic.
+  - Any other related files (e.g., constants, types, utilities)
+
+  Example directory structure:
+
+  ```
+  src/
+  └── patient-banner/
+      ├── patient-banner.component.tsx
+      ├── patient-banner.test.tsx
+      ├── patient-banner.scss
+      └── patient-banner.resource.ts
+  ```
+
+  This organization:
+  - Makes it easier to find and modify related files
+  - Simplifies refactoring and maintenance
+  - Keeps the codebase modular and well-organized
+  - Makes it clear which files belong to which component
+
+- Avoid placing styles for multiple components in the same stylesheet. Instead, create a separate stylesheet for each component. This makes it easier to find the styles for a particular component.
```

Comment:
It definitely is indicative of something wrong. The idea of having per-component styles is really about component hygiene, i.e., making sure each component is its own, composable unit. It may be valid to restyle a component in the context of being deployed in another component, but that's better done by allowing the restyled component to, e.g., take classes as an argument or that kind of thing.